add not having a single, dedicated PO as a risk

suprenant opened the pull request:
Flagging this as a risk to help us name this and resolve soon.

Commit:
add not having a single, dedicated PO as a risk

Flagging this as a risk to help us name this and resolve soon.

tbaxter-18f commented:
I'd suggest, given the pace of this and the number of open questions, that we may want to ID both primary and secondary points-of-contact. There will definitely be times when a single primary contact will be away or otherwise engaged.

suprenant commented:
@tbaxter-18f makes sense. So I'm clear, are you suggesting we make this update here? That's not a bad idea. Feel free to propose an edit accordingly, or let me know if you want me to change it.

tbaxter-18f commented on README.md:
```diff
@@ -91,6 +91,9 @@ We will assist NAVAIR with modern data practices aimed at a comprehensive unders
 
 ## Risks and mitigations
 
+**Risk:** We do not designate a single product owner who is available to make decisions, support this work, provide resources, and work to unblock issues in a timely manner.  
```
Perhaps something like:

**Risk:** We do not have designated product owners available to make decisions, support ... etc.
**Mitigation:** Identify and select primary and secondary points of contact who can provide this support, with the secondary contact as a fallback mechanism when the primary is unavailable.

Commit:
Update README.md

Your feedback looks good to me @tbaxter-18f, can you please re-review? 

If it looks good, please feel free to approve, merge, and delete the old branch.
